.travis.yml

```diff
@@ -1,61 +1,20 @@
-sudo: required
-dist: trusty
-
+dist: bionic
 language: go
-go_import_path: github.com/coreos/ignition
```

Comment:
I think this would be useful to keep, for the usecase where people clone this repo to their own account and run travis there (i.e. forks under different orga/import-path).

Reply:
:+1: updated to be correct (needs a v2). I don't think it actually matters since we're building with our own scripts, but I'll leave it in for people that fork and want to do something different.